packages/context/src/objects/objects.ts

```diff
@@ -43,6 +43,37 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
+export type FormioInputTypes = {
+    label: string;
+    type: string;
+    key: string;
+    initialValue?: any;
+    property?: Property;
+    input?: boolean;
+    placeholder?: string;
+    description?: string;
+    tooltip?: string;
+    prefix?: string;
+    suffix?: string;
+    showCharCount?: boolean;
+    readOnly?: boolean;
+    isMultiLineText?: boolean;
+    widget?: string;
+    tableView?: boolean;
+    verticalLayout?: boolean;
+    data?: {
+        resource?: string;
+        values?: [];
+    };
+    dataSrc?: string;
+    validate?: {
+        required: boolean;
+        operator: any;
+    };
+    components: FormioInputTypes[];
```

Comment:
Is this for the Section layout field type?  Maybe it's required for sections, but in the grand scheme, I'd think this should be marked optional, or we need more intricate typings to show that it's only required for sections.

Can you also add a tsdoc here to explain what this is?

Reply:
Done.